docs/articles/preamble.md

```diff
@@ -0,0 +1,43 @@
+---
+uid: preamble
+title: Article Preamble
+---
+
+## Knowledge Prerequisites 
+Before attempting to write a Discord bot, you should be familiar with the concepts of [Object Oriented Programing](https://en.wikipedia.org/wiki/Object-oriented_programming), [the C# programming language](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/), and [Task-based Asynchronous Pattern](https://docs.microsoft.com/en-us/dotnet/standard/asynchronous-programming-patterns/task-based-asynchronous-pattern-tap).
+
+If you're brand new to C#, or programming in general, this library may prove difficult for you to use.</br>
+Fortunately, there are resources that can help you get started with the language! 
+
+An exellent tutorial series to go through would be [C# Fundamentals for Absolute Beginners](https://channel9.msdn.com/Series/C-Fundamentals-for-Absolute-Beginners) by Bob Tabor.
+His videos go through all the basics, from setting up your development environment up to some of the more advanced concepts. 
+If you're not sure what to do first, Bob's tutorial series should be your starting point!
+
+## Supported .NET Implementations
+Because DSharpPlus targets .NET Standard 2.0, there are many implementations that may function with DSharpPlus.
+However, there are only a few versions we will *explicitly* provide support for.
+
+Implementation|Support|Notes
+:---: |:---:|:---
+[.NET Core](https://en.wikipedia.org/wiki/.NET_Core)|✔️|LTS versions 2.1 and 3.1 are supported.
+[.NET Framework](https://en.wikipedia.org/wiki/.NET_Framework)|⚠️|Versions 4.6.1 through 4.8 *should* work fine.<br/>However, we do not directly support .NET Framework.<br/>We recommend that you use the latest LTS version of .NET Core.
+[Mono](https://en.wikipedia.org/wiki/Mono_(software))|❌️|Has numerous flaws which can break things without warning.<br/>If you need a cross platform runtime, use .NET Core.
+[Unity](https://en.wikipedia.org/wiki/Unity_(game_engine))|❌️|Despite having support for C#, Unity is a game engine.<br/>As such, it will *never* be supported by DSharpPlus.
```

Comment:
Perhaps suggest GameSDK here? It is the correct way of integrating games with Discord.

Definitely should be mentioned that embedding bots in a game is a surefire way of leaking credentials, and should be avoided, and, besides, is not a correct use case in the first place.